tasks/clone_foundation_site/task.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+
+# Clone and scrub the production foundation site
+
+on_exit() {
+    RV=$?
+
+    # If the script exits with a non-zero exit code, force a staging rollback and
+    # ensure that staging is not in maintenance mode or scaled down
+    if [ $RV -ne 0 ]; then
+        echo "Rolling back staging..."
+        heroku pg:backups:restore -a ${staging_app} --confirm ${staging_app}
+
+        echo "Scaling web dynos on staging to 1..."
+        heroku ps:scale -a ${staging_app} web=1
+
+        echo "Disabling maintenance mode on the staging app..."
+        heroku maintenance:off -a ${staging_app}
+    else
+        echo "task complete!"
+    fi
+
+    exit $RV
+}
+
+trap 'on_exit' 0
+
+set -e
+
+current_dir=$(pwd)
+
+echo "Checking the day of the week..."
+DAYOFWEEK=$(date +%u)
+if [ "${DAYOFWEEK}" -ne 4 ]; then
```

Comment:
Shouldn't it be 1 instead of 4?

Reply:
heh, we had set it to 4 because we were testing it on Thursday =P

Reply:
I realized the typo when I had to put it to 5 to do testing on Friday XD